app/templates/views/privacy.html

```diff
@@ -63,12 +63,11 @@ <h2 class="heading-medium">How long we keep your data</h2>
 
     <h2 class="heading-medium">Where your data is processed and stored</h2>
 
-    <p class="govuk-body">We design, build and run our systems to make sure that your data is as safe as possible at any stage, both while it’s
+    <p class="govuk-body">We design, build and run our systems to make sure that your data is as safe as possible at every stage, both while it’s
       processed and when it’s stored.</p>
 
-    <p class="govuk-body">Your personal data will not be transferred outside of the European Economic Area (EEA) throughout the course of its
-      processing at GDS or any third party suppliers.</p>
-
+    <p class="govuk-body">Your personal data will be processed both in the United Kingdom and the European Economic Area (EEA). Your data receives the same level of protection in the EEA as it does in the UK through the safeguard of <a class="govuk-link govuk-link--no-visited-state" href="https://ico.org.uk/for-organisations/dp-at-the-end-of-the-transition-period/data-protection-and-the-eu-in-detail/adequacy/">Adequacy Decisions</a>.</p>
```

Comment:
```suggestion
    <p class="govuk-body">Your personal data will be processed both in the UK and the European Economic Area (EEA). Your data receives the same level of protection in the EEA as it does in the UK through the safeguard of <a class="govuk-link govuk-link--no-visited-state" href="https://ico.org.uk/for-organisations/dp-at-the-end-of-the-transition-period/data-protection-and-the-eu-in-detail/adequacy/">Adequacy Decisions</a>.</p>
```

I don’t think we need to write ‘United Kingdom’ in full.